.env

```diff
@@ -9,6 +9,8 @@ COLLECTOR_ZIPKIN_HOST_PORT=:9411
 OTEL_EXPORTER_OTLP_TRACES_ENDPOINT=http://otelcol:4317
 OTEL_EXPORTER_OTLP_ENDPOINT=${OTEL_EXPORTER_OTLP_TRACES_ENDPOINT}
 
```

Comment:
Curious, any reason to leave a blank here? Doesn't seem to be consistent with Ln9-10.

Reply:
no reason. Removing it